substrate/frame/multisig/src/lib.rs

```diff
@@ -176,14 +184,26 @@ pub mod pallet {
 	pub struct Pallet<T>(_);
 
 	/// The set of open multisig operations.
+	#[deprecated = "MultisigsFor"]
 	#[pallet::storage]
 	pub type Multisigs<T: Config> = StorageDoubleMap<
 		_,
 		Twox64Concat,
 		T::AccountId,
 		Blake2_128Concat,
 		[u8; 32],
-		Multisig<BlockNumberFor<T>, BalanceOf<T>, T::AccountId, T::MaxSignatories>,
+		OldMultisig<BlockNumberFor<T>, BalanceOf<T>, T::AccountId, T::MaxSignatories>,
+	>;
+
+	/// The set of open multisig operations.
+	#[pallet::storage]
+	pub type MultisigsFor<T: Config> = StorageDoubleMap<
```

Comment:
This is a breaking change and will probably mess with wallets/indexers etc since they look at the old map.  
I dont think there is an easy way out without MBMs (audit stalled for another month), so not sure what to best do.

Reply:
TBH since this is not the highest priority I don't think too much of a problem to wait for MBMs.. should try to minimise breakage of these apis as much as possible, as you point out. with this change, possibly every multisig UI in the ecosystem would need updating :P

Reply:
Yeah, fine to wait for MBM if that's the case.